dash/development/_r_components_generation.py

```diff
@@ -509,7 +539,17 @@ def generate_rpkg(
 
     package_name = snake_case_to_camel_case(project_shortname)
     lib_name = pkg_data.get("name")
-    package_description = pkg_data.get("description", "")
+
+    if rpkg_data is not None:
+        if rpkg_data.get("pkg_help_title"):
+            package_title = rpkg_data.get("pkg_help_title",
+                                          pkg_data.get("description",
+                                                       ""))
```

Comment:
Is the extra return on purpose?

Generating for the table I now get extra empty lines:
```
Package: dashTable
Title: Core Interactive Table Component for Dash

Version: 4.5.1
Description: Dash DataTable is an interactive table component designed for designed for viewing, editing, and exploring large datasets. DataTable is rendered with standard, semantic HTML <table/> markup, which makes it accessible, responsive, and easy to style. This component was written from scratch in React.js specifically for the Dash community. Its API was designed to be ergonomic and its behaviour is completely customizable through its properties.

Depends: R (>= 3.0.2)
Imports: 
Suggests: 
License: MIT + file LICENSE
URL: https://github.com/plotly/dash-table
BugReports: https://github.com/plotly/dash-table/issues
Encoding: UTF-8
LazyData: true
Author: Chris Parmer [aut]
Maintainer: Ryan Patrick Kyle <ryan@plot.ly>

```

Reply:
The funky formatting of the lines from 544:547 is to satisfy the linting overlords.

I'm not seeing the extra linefeeds you have above, but that's probably because I edited my `dash-info.yaml` files such that the `>` operator has been replaced with the "chomping" operator `>-`. I've since back-committed them to `dev` via new PRs, so hopefully you'll generate the same `DESCRIPTION` I have committed to this and the other two core component repositories.